### PR TITLE
Update rpath injection hook for `versions` subdir

### PR DESCRIPTION
**Pull request by ocaisa**

### CODE REVIEW

File: eb_hooks.py [hunk 1]
```diff
@@ -27,12 +27,11 @@ def get_eessi_envvar(eessi_envvar):
 def get_rpath_override_dirs(software_name):
     # determine path to installations in software layer via $EESSI_SOFTWARE_PATH
     eessi_software_path = get_eessi_envvar('EESSI_SOFTWARE_PATH')
-    eessi_pilot_version = get_eessi_envvar('EESSI_PILOT_VERSION')
 
     # construct the rpath override directory stub
     rpath_injection_stub = os.path.join(
         # Make sure we are looking inside the `host_injections` directory
-        eessi_software_path.replace(eessi_pilot_version, os.path.join('host_injections', eessi_pilot_version), 1),
+        eessi_software_path.replace('versions', 'host_injections', 1),
```
Comment (boegel) [inline eb_hooks.py:34]: Did you mean `versions` / `eessi_pilot_version` ?

**Review by boegel — CHANGES_REQUESTED** [review]

File: eb_hooks.py (hunk 1 above)
Comment (ocaisa) [reply eb_hooks.py:34]: I meant `versions`:
```
[EESSI pilot 2021.12] $ echo $EESSI_SOFTWARE_PATH 
/cvmfs/pilot.eessi-hpc.org/versions/2021.12/software/linux/aarch64/graviton2
```

Comment (ocaisa) [reply eb_hooks.py:34]: It's a swap `versions` -> `host_injections`, so the final path contains:
```
/cvmfs/pilot.eessi-hpc.org/host_injections/2021.12/software/linux/aarch64/graviton2
```

Comment (boegel) [reply eb_hooks.py:34]: Right, ok, thanks for clarifying

**Pull request merged**